Gemfile

```diff
@@ -13,7 +13,7 @@ group :preload do
   # AR extensions
   gem 'goldiloader'
   gem 'pagy'
-  gem 'audited'
+  gem 'audited', '~>5.4.0'
```

Comment:
either remove the requirement when you are done or leave a `# 5.5 is broken <issue link>` kind of note

Reply:
Yup, just wanted to work through versions systematically. Definitely plan to remove once I'm done upgrading.